src/shared/lib/accounts.ts

```diff
@@ -17,7 +17,7 @@ import {
 type TrendEntry = {
   amount: number;
   date: string;
-  type: string;
+  type: 'DEBT' | 'ASSET' | 'EXPENSE' | string;
```

Comment:
Any idea what EXPENSE is and whether it also needs inverting?

Reply:
It's one of the types of reports Mint returns (SPENDING_TIME).

Right now, we only use the reports ASSETS_TIME and DEBTS_TIME to download the balances, and they should return either DEBT or ASSET. I'll remove this EXPENSE one to avoid any confusion.